client/src/pages/CreateWorkout/components/ExerciseCard/ExerciseCard.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import { MdDeleteForever } from "react-icons/md";
+import { MdModeEdit } from "react-icons/md";
+
+const ExerciseCard = ({
+  exercise,
+  handleShowForm,
```

Comment:
One thing I'd like to challenge is to think about whether your exercise card & exercise form really need to have the `handleShowForm` or even know about whether or not a form is showing?

I.e. the parent of ExerciseCard should determine whether or not when edit is clicked, if it shows a form, redirects, or does something else entirely. ExerciseCard should just care about telling the parent "hey my edit button was clicked" by calling a "handleEdit" callback that's passed in. That way -- this card doesn't need to know to combine handleShowForm & handleEditExercise together.

Reply:
Similarly, your `ExerciseForm` below is really just calling handleShowForm to tell it to close itself - which it could take in a function to tell the parent "hey my close button  was clicked" so instead of taking in handleShowForm it could take in something more like `handleClose` which in the parent is defined as `handleShowForm(false)`